### PR TITLE
For #6295: Clear focus for username/password when Done is pressed

### PR DESCRIPTION
**Pull request by ValentinTimisica**
---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks.
- [x] **Tests**: This PR does not include thorough tests.
- [x] **Changelog**: This PR or does not need [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md).
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md).

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/feature/prompts/src/main/java/mozilla/components/feature/prompts/ext/EditText.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import android.view.inputmethod.EditorInfo
+import android.widget.EditText
+
+/**
+ * Extension function to handle keyboard Done action
+ * @param actionConsumed true if you have consumed the action, else false.
+ * @param onDonePressed callback to execute when Done key is pressed
+ */
+internal fun EditText.onDone(actionConsumed: Boolean, onDonePressed: () -> Unit) {
```
Comment (psymoon) [inline components/feature/prompts/src/main/java/mozilla/components/feature/prompts/ext/EditText.kt]: Can we add a test for this?

Comment (ValentinTimisica) [reply components/feature/prompts/src/main/java/mozilla/components/feature/prompts/ext/EditText.kt]: Sure. I added now. Thanks!

**Comment by ValentinTimisica** [comment]
Thanks! Do you know if I have access to `bors r+` command?

**Comment by ValentinTimisica** [comment]
bors r=psymoon

**Pull request merged**